Dockerfile

```
@@ -6,9 +6,8 @@ COPY . .
 
 RUN npm install
 RUN npm link --local
-RUN mkdir /data
 
 EXPOSE 3000
-VOLUME /data
+VOLUME ["/data"]
```

```suggestion
VOLUME /data
```

Why is this in an array?